### PR TITLE
Implement post multitag datapoints

### PR DESCRIPTION
**Pull request by stianlagstad**
Would like some feedback on this before I do the same with v0.5.

### CODE REVIEW

File: cognite/v04/timeseries.py [hunk 1]
```diff
@@ -191,6 +191,81 @@ def _get_datapoints_helper(tag_id, aggregates=None, granularity=None, start=None
...
+        list_items=timeseries_with_datapoints_limited,
+        max_size=ul_dps_limit,
+        get_count=lambda x: len(x.datapoints)
+    )
+
+    for bin in timeseries_to_upload_binned:
+        body = {
+            'items': [ts_with_data.__dict__ for ts_with_data in bin]
+        }
+        res = _utils.post_request(url, body=body, headers=headers)
```
Comment (KennethSkaar) [inline cognite/v04/timeseries.py]: Shouldn't you check that the result here is OK for each request? Or will it throw an exception on failure?

Comment (stianlagstad) [reply cognite/v04/timeseries.py]: I definitely should! Will look at that now.

Comment (stianlagstad) [reply cognite/v04/timeseries.py]: @KennethSkaar: Shouldn't the logic within `_utils.post_request` be enough? If an apierror is thrown from there, then the loop here won't continue

Comment (KennethSkaar) [reply cognite/v04/timeseries.py]: true, looks like it does the right thing

File: cognite/_utils.py [hunk 2]
```diff
@@ -168,3 +169,64 @@ def __init__(self, tag_ids):
...
+
+
+class Bin:
+    entries: []
+    get_count: Callable
+
+    def __init__(self, get_count):
+        '''
+        :param get_count: A function that will take an element and get the count of something in it.
+        '''
```
Comment (erlendcognite) [inline cognite/_utils.py]: Google style docstrings

**Pull request merged**